go/kubebuilder/memcached-operator/Makefile

```diff
@@ -50,12 +50,6 @@ deploy: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
-# UnDeploy controller from the configured Kubernetes cluster in ~/.kube/config
-# Note that it was added for we are allowed to uninstall the files. However, it will be present by default in the future
-# versions.
-undeploy:
-	$(KUSTOMIZE) build config/default | kubectl delete -f -
-
```

Comment:
This one we need to append at the end of the file as well.

Reply:
Added at the end of Makefile